features/hsl.yml

```
@@ -0,0 +1,12 @@
+name: HSL
+description: The `hsl()` and `hsla()` CSS functions pick colors using hue, saturation and lightness channels.
```

Let's mention the alpha channel too:

```suggestion
description: The `hsl()` and `hsla()` CSS functions pick colors using hue, saturation, lightness, and alpha (transparency) channels.
```